zipkin-server/README.md

```diff
@@ -113,98 +111,110 @@ exposition [text format version 0.0.4](https://prometheus.io/docs/instrumenting/
 
 Collector metrics are broken down by transport. The following are exported to the "/metrics" endpoint:
 
-Metric | Description
---- | ---
-counter.zipkin_collector.messages.$transport | cumulative messages received; should relate to messages reported by instrumented apps
-counter.zipkin_collector.messages_dropped.$transport | cumulative messages dropped; reasons include client disconnects or malformed content
-counter.zipkin_collector.bytes.$transport | cumulative message bytes
-counter.zipkin_collector.spans.$transport | cumulative spans read; should relate to messages reported by instrumented apps
-counter.zipkin_collector.spans_dropped.$transport | cumulative spans dropped; reasons include sampling or storage failures
-gauge.zipkin_collector.message_spans.$transport | last count of spans in a message
-gauge.zipkin_collector.message_bytes.$transport | last count of bytes in a message
+Metric | Tags | Description
+--- | --- | ---
+trace_in_latency_bucket_bucket | sw_backend_instance=$instance_host_$instance_port, protocol=$transport, le=$second_bucket | The process latency histogram of trace data
+trace_in_latency_sum | sw_backend_instance=$instance_host_$instance_port, protocol=$transport | The total process latency of trace data
+trace_analysis_error_count | sw_backend_instance=$instance_host_$instance_port, protocol=$transport | The error number of trace analysis
 
 ## Configuration
-We support ENV variable configuration, such as `STORAGE_TYPE=cassandra3`, as they are familiar to
+We support ENV variable configuration, such as `ZIPKIN_STORAGE=cassandra`, as they are familiar to
```

Comment:
```suggestion
We support ENV variable configuration, such as `ZIPKIN_STORAGE=cassandra3`, as they are familiar to
```

I think we consistent with previous. Could you rechecck?

Reply:
Update the names to `cassandra3`